esp/services/ws_sql/ws_sqlService.cpp

```diff
@@ -1041,9 +1086,16 @@ bool CwssqlEx::onExecuteSQL(IEspContext &context, IEspExecuteSQLRequest &req, IE
             int timeToWait = req.getWait();
             if (timeToWait != 0)
             {
-                context.addTraceSummaryTimeStamp(LogNormal, "StartWUProcessWait");
-                waitForWorkUnitToComplete(runningwuid.str(), timeToWait);
-                context.addTraceSummaryTimeStamp(LogNormal, "EndWUProcessWait");
+                OwnedSpanScope wuProcessSpanScope(queryThreadedActiveSpan()->createInternalSpan("wait_for_workunit_to_complete"));
+                try
+                {
+                    waitForWorkUnitToComplete(runningwuid.str(), timeToWait);
```

Comment:
I'm not sure where this function is defined, but seems worth tracing it (perhaps outside the scope of this PR)

Reply:
@rpastrana the method is defined in workunit.cpp. It redirects through a factory to access one of multiple implementations (one of which is "unimplemented"). The method is called from multiple services which are not instrumented with tracing. Even within this service, only the calls being made within traced methods are included in child spans.

I think a separate PR is appropriate for changing the common function. Changes in this service can be made as part of this PR after additional discussion.